.github/workflows/updates/LineRider.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+webVer=$(get_release Sussy-OS/LRA-Community-Edition)
+all_url="https://github.com/Sussy-OS/LRA-Community-Edition/releases/download/${webVer}/LineRider.game.auto-release.${webVer}-LR.zip"
+pi_apps_ver="$(cat 'install' | grep 'version=' | sed 's/version=//')"
```

Comment:
```suggestion
```

You don't need to specify `pi_apps_ver` in the script. The `update_github_script.sh` will do that for you.

Reply:
thx